### DISCUSSION TITLE
bug login expired

### DISCUSSION THREAD

**Discussion by MattMello88** [filamentphp/filament#12922] [Help]
### Package

filament/filament

### Package Version

v3x

### Laravel Version

v11

### PHP Version

PHP 8.2

### Problem description

When the session expires and I return with the same login, ok, but when the session expires and I return with another user, the login goes to an expired page, it only returns if I change the url

### Expected behavior

Validate that there was a user change at login when the session expired and clear the browsing history

### Steps to reproduce

Login with first user and let it expire session and then login with second user.

### Reproduction repository

https://github.com/pet-labs/pet-app/tree/feature-tenancy

**Comment by danharrin** [comment]
No public reproduction repository, and probably nothing we can do about a session expiring
